docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+---
+date: 2021-03-31
+title: "Backing up K8ssandra with MinIO"
+linkTitle: "Backing up K8ssandra with MinIO"
+description: >
+Learn how to set up K8ssandra and MinIO to backup Cassandra in just a few steps.
+author: Alexander Dejanovski ([@alexanderDeja](https://twitter.com/alexanderDeja))
+draft: true
+---
+
+# Backing up K8ssandra with MinIO
+
+K8ssandra includes Medusa for Apache Cassandra™ to handle backup and restore for your Cassandra nodes. Recently Medusa was upgraded to introduce support for all S3 compatible backends, including [MinIO](https://min.io/), the popular k8s-native object storage suite. Let’s see how to set up K8ssandra and MinIO to backup Cassandra in just a few steps.
+
+# Deploy MinIO
+
+Similar to K8ssandra, MinIO can be simply deployed through Helm.
+
+First, add the MinIO repository to your local list:
+
+```
+helm repo add minio https://helm.min.io/
+```
+
+The MinIO Helm charts allow you to do several things at once at install time:
+
+*   Set the credentials to access MinIO
+*   Create a bucket for your backups that can be set as default
+
+You can create a **k8ssandra-medusa** bucket and use **minio_key/minio_secret** as the credentials, and deploy MinIO in a new namespace called **minio** by running the following command:
+
+```
+helm install --set accessKey=minio_key,secretKey=minio_secret,defaultBucket.enabled=true,defaultBucket.name=k8ssandra-medusa minio minio/minio -n minio --create-namespace
+```
+
+**Note:** Creating the bucket is not mandatory at this stage and can be done through MinIO’s UI.
+
+After the `helm install` command has completed, you should see something similar to this in the **minio** namespace: 
+
+```
+% kubectl get all -n minio
+NAME                        READY   STATUS    RESTARTS   AGE
+pod/minio-5fd4dd687-gzr8j   1/1     Running   0          109s
+
+NAME            TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/minio   ClusterIP   10.96.144.61   <none>        9000/TCP   109s
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/minio   1/1     1            1           109s
+
+NAME                              DESIRED   CURRENT   READY   AGE
+replicaset.apps/minio-5fd4dd687   1         1         1       109s
+```
+
+Using port forwarding, you can expose access to the MinIO UI in the browser on port 9000:
+
+```
+% kubectl port-forward service/minio 9000:9000 -n minio
```

Comment:
This can be shortened to

```suggestion
% kubectl port-forward service/minio 9000 -n minio
```

Reply:
Note this suggestion is merely a nit and not a blocker.

Reply:
neat, didn't know that 👍